### ISSUE TITLE
USB PID

### ISSUE THREAD

**Issue by dominicgs** [rad1o/hackrf#3]
Commit fb219b6 ("Let the rad1o be a hack-rf on the usb for now") uses the HackRF One USB PID for rad1o.  I assume that this was to make it compatible with libhackrf.  Is that correct?

We think it's probably best if the two hardware variants have different PIDs so that we can handle them differently.  This will make it easier for libhackrf to support rad1o hardware and for us to continue supporting libhackrf.

In order to maintain compatibility between libhackrf and rad1o, we would also like to pull the relevant changes upstream, e.g. the firmware changes for rad1o.

Does this fit with your plans?
Would you like our help with this task?

**Comment by Sec42** [comment]
Yes, that change was made for compatibility reasons, so i "just works" with libhackrf/gnuradio for now.

From my POV there is nothing against changing the PID to something different, if the "standard" libhackrf would then support it.

I see no problems in you merging the relevant changes -- But be warned, that those changes are currently incomplete (there are at least two bugs known to me that need to be fixed)

If you need anything specific from us, we are willing to help.

**Comment by dominicgs** [comment]
We plan to make a release of the HackRF tools in the next few weeks.  I'd like to support rad1o in libhackrf with that release.

When it comes to the firmware changes, I think it's best that you continue to have the rad1o firmware in your repository for now and we can take a look at merging appropriate things later.

I'll probably need your help to identify what is relevant.  The easiest way for me to do this is probably to join your IRC channel.

**Comment by Sec42** [comment]
you probably need to talk to schneider about this, as he made the hackrf firmware patches. I agree that the IRC is probably the best way, and I sent you the contact data privately yesterday, but in case you missed it, check https://rad1o.badge.events.ccc.de/contact for it.
